### PR TITLE
 [Term Entry] Java Calendar getTimeZone() #3098 

### LINKED ISSUE #3098: [Term Entry] Java Calendar getTimeZone()
### Reason/inspiration (optional)

Please add the `getTimeZone()` method as a new term entry for the [Calendar](https://www.codecademy.com/resources/docs/java/calendar)�class. Provide the entry in a markdown file and place under the file path `content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md`

The entry should have the following:

�  An introductory paragraph about what the method is and does.
�  A syntax section with a pseudo code block with an explanation of it.
�  An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by HamishHill-WK**
### Description

- Added `getTimeZone` directory to `content/java/concepts/calendar/terms/getTimeZone`

- Added new term entry file to `content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md` for Java Calendar Class
Closes  #3098 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:3]:
```suggestion
Description: 'Returns the current time-zone of a given Calendar.'
```

File: content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:17]:
```suggestion
The **`.getTimeZone()`** method in Calendar class is used to return the current time-zone of a given Calendar.
```

File: content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:26]: We want a introductory sentence to summarize what the example is doing.

File: content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:24]:
```suggestion
We want a sentence after the syntax block explaining it. i.e. what `myCalendar` and `time_zone` are.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
I did a first review and left a few comments. Please note that you'll need to change your PR so it isn't a draft, otherwise we cannot merge it.

File: content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:24]:
```suggestion

- `myCalendar` is a `Calendar` class object for which the time zone is being retrieved.. 
- `time_zone` is a `TimeZone` class object returned from the `.getTimeZone()` method.
```

This is all we need for the syntax section.

File: content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:26]:
```suggestion
## Example

The following code creates a Calendar class object and then assigns the object's current time zone to a TimeZone object. The result is then printed to the screen. 
```
You deleted the example header.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@HamishHill-WK Two more comments.

**Review by SSwiniarski — APPROVED** [review]
@HamishHill-WK Looks good for a second review.

File: content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:16]:
```suggestion
The **`.getTimeZone()`** method in the `Calendar` class is used to return the current time-zone of a given calendar.
```

File: content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:24]:
```suggestion
- `myCalendar` is a `Calendar` class object for which the time zone is being retrieved.
```

File: content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:21]:
```suggestion
time_zone = myCalendar.getTimeZone();
```

File: content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:29]:
```suggestion
The following code creates demonstrates the use of the `.getTimeZone()` method. 
```

File: content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:31]: update the code to use 2 spaces for indentations. Tabs are not used in docs

File: content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:51]:
```suggestion
The above code will return an output similar to:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@HamishHill-WK I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by HamishHill-WK** [comment]
@caupolicandiaz thank you for your review, I've updated the code example to use spaces instead of tabs and implemented the other changes you requested

**Review by caupolicandiaz — APPROVED** [review]
@HamishHill-WK @SSwiniarski LGTM

**Pull request merged**
